src/_repobee/command/repos.py

```diff
@@ -226,7 +222,7 @@ def _create_push_tuples(
             api=api,
         )
 
-        yield created, git.Push(
+        yield created, Push(
```

Comment:
I also changed this to `Push` so that a developer is not confused between `git.push` and `git.Push`.

Reply:
`git.Push` is a really bad name. Really, the whole `git` module suffers from some really confusing naming. It needs some refactoring.